release/release.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 set -uex
 
+unset $(env | cut -d= -f1 | grep -v '^HOME$' | grep -v '^PWD$' | grep -v '^USER$' | grep -v '^SHELL$' | grep -v '^TERM$')
```

Comment:
```suggestion
unset $(env | cut -d= -f1 | grep -Fx HOME PWD USER SHELL TERM)
```
but actually `unset` doesn't require the variables to be set at all, so isn't:
```suggestion
unset HOME PWD USER SHELL TERM
```
OK?

Reply:
Oh `-v`! Sorry... it is the `grep`, form but `unset $(env | cut -d= -f1 | grep -Fvx HOME PWD USER SHELL TERM)`!